api/lib/domain/services/badge-criteria-service.js

```diff
@@ -1,9 +1,10 @@
 const _ = require('lodash');
 const BadgeCriterion = require('../../../lib/domain/models/BadgeCriterion');
 
-function areBadgeCriteriaFulfilled({ campaignParticipationResult, badgeCriteria }) {
-  return _.every(badgeCriteria, (criterion) => {
-    let isBadgeCriterionFulfilled;
+function areBadgeCriteriaFulfilled({ campaignParticipationResult, badge }) {
+  return _.every(badge.badgeCriteria, (criterion) => {
+    let isBadgeCriterionFulfilled = false;
+    let campaignParticipationBadge;
```

Comment:
Pour simplifier, est-ce qu'il est possible de faire ça ? 

```suggestion
function areBadgeCriteriaFulfilled({ campaignParticipationResult }) {
  const badges = campaignParticipationResult.campaignParticipationBadges;
  
  return _.every(badge.badgeCriteria, (criterion) => {
    let isBadgeCriterionFulfilled = false;
    let campaignParticipationBadge;
```